test/extended/prometheus/prometheus_builds.go

```diff
@@ -21,7 +21,7 @@ var (
 	statsPort                          int
 )
 
-var _ = g.Describe("[Feature:Prometheus][Feature:Builds] Prometheus", func() {
+var _ = g.Describe("[Feature:Prometheus][Feature:Builds] Prometheus-GGM", func() {
```

Comment:
s/-GGM//

Reply:
dammit ... meant to remove that - thx

Reply:
we have a strict "no vanity test case names" rule around here....  :)